test/mocks/navigation/route_mocks.dart

```diff
@@ -0,0 +1,253 @@
+import 'package:flutter/material.dart';
```

Comment:
I made a stub for the test in `navigation_service_test.dart` - the only functions that do anything but throw UnimplementedError are the first two getters